tutorindigo/templates/indigo/lms/templates/footer.html

```diff
@@ -137,6 +128,25 @@
             
         </ol>
       </nav>
+      <div class="footer-column footer-social" aria-label="${_('Follow us on social media')}">
+        <ul class="social-icons">
+            <li>
+                <a href="https://x.com/uahes" target="_blank" rel="noopener" aria-label="Twitter">
+                    <i class="fa-brands fa-x-twitter" aria-hidden="true"></i>
+                </a>
+            </li>
+            <li>
+                <a href="https://www.instagram.com/uahes" target="_blank" rel="noopener" aria-label="Instagram">
+                    <i class="fa-brands fa-instagram" aria-hidden="true"></i>
+                </a>
+            </li>
+            <li>
+                <a href="https://www.facebook.com/UniversidadDeAlcala/" target="_blank" rel="noopener" aria-label="Facebook">
+                    <i class="fa-brands fa-facebook" aria-hidden="true"></i>
+                </a>
+            </li>
+        </ul>
+    </div>
     </div>
     <span class="copyright-site">2024 © Universidad de Alcal&aacute;.</span>
     % endif
```

Comment:
The date should be 2024 or 2025? curious about it because the change above is 2025 but here is 2024

Reply:
Fixed! I updated to 2025.